Dockerfile

```diff
@@ -1,61 +1,94 @@
 FROM fedora:31
 
-# Bug https://bugzilla.redhat.com/show_bug.cgi?id=1694411
-RUN echo "zchunk=False" >> /etc/dnf/dnf.conf && \
-    dnf install -y git gcc httpd python3 python3-pip python3-devel \
-        python3-mod_wsgi mod_auth_openidc libffi-devel openssl-devel
+RUN dnf install -y \
+	findutils \
+	gcc \
+	git \
```

Comment:
Is there anything else that still needs git?

Reply:
There's not, but I figured it couldn't hurt to have it in the image. I don't feel strongly about this one.